cmd/winch/main.go

```diff
@@ -46,6 +46,7 @@ import (
 
 var (
 	flagHttpPort              = sharedflags.Set.Int("server_http_port", 8070, "TCP port to listen on for HTTP1.1/REST calls.")
+	flagMetricsPath           = sharedflags.Set.String("server_metrics_path", "/metrics", "path on which to serve metrics")
```

Comment:
This should be `/_metrics` for backwards compatibility

Reply:
Disagree. The default Prometheus path is `/metrics`. Also this makes me thinking that I need to change kedge default value too.

From our infra perspective, we change it explicitly 100% time so we are fine.